test/integration/scheduler/podsready/scheduler_test.go

```diff
@@ -293,6 +294,9 @@ var _ = ginkgo.Describe("SchedulerWithWaitForPodsReady", func() {
 				})
 				g.Expect(k8sClient.Status().Update(ctx, prodWl)).Should(gomega.Succeed())
 			}, util.Timeout, util.Interval).Should(gomega.Succeed(), "Job reconciler should add an Evicted condition with InactiveWorkload to the Workload")
+
+			// ReconcileGenericJob didn't run here
+			// util.ExpectEvictedWorkloadsTotalMetric(prodClusterQ.Name, kueue.WorkloadEvictedByDeactivation, 1)
```

Comment:
I think this requires some investigation, or explanation why we don't see the metric bumped.

Reply:
l looked closer into this, and it appears to be fixed here: https://github.com/kubernetes-sigs/kueue/pull/2131/. The metric should also be bumped into the new section in the `workload_controller` now. 

Additionally, IIUC we can now remove the block above, please do so: `TODO: Once we move a logic to issue the Eviction with InactiveWorkload reason, we need to remove the below updates.`.

Reply:
> l looked closer into this, and it appears to be fixed here: #2131. The metric should also be bumped into the new section in the `workload_controller` now.

Done.